Source/gg2/Scripts/Plugins/checkpluginname.gml

```diff
@@ -10,9 +10,9 @@ if (string_length(argument0) < 1)
 }
 
 validChars = "0123456789abcdefghijklmnopqrstuvwxyz_";
-for (i = 0; i < string_length(argument0); i+=1)
+for (i = 1; i <= string_length(argument0); i+=1)

```

Comment:
So does that mean we always forgot to check the last character until now?

Reply:
Yes. Strings in GM always were 1-based.